alembic/runtime/migration.py

```diff
@@ -676,13 +676,14 @@ def _delete_version(self, version):
                 == literal_column("'%s'" % version)
             )
         )
-        if not self.context.as_sql and ret.rowcount != 1:
-            raise util.CommandError(
-                "Online migration expected to match one "
-                "row when deleting '%s' in '%s'; "
-                "%d found"
-                % (version, self.context.version_table, ret.rowcount)
-            )
+        if not self.context.as_sql and ret.supports_sane_rowcount():
```

Comment:
this should all be one "if" statement.  if you use the pre-commit hooks to do the "black" formatting it will format it consistently, or just use "black -l79 alembic/runtime/migration.py".

Reply:
Thanks!